apps/src/aichat/views/chatMessage.module.scss

```diff
@@ -45,7 +45,7 @@
   background-color: $light_gray_50;
   border-radius: 0 16px 16px 16px;
   margin-top: 14px; // a quarter of height of bot icon
-  overflow-x: scroll;
+  overflow-x: auto;
```

Comment:
Do we want to allow horizontal scrolling at all actually?

Reply:
Yes, there's a case where content was overflowing but wasn't scrollable (tbh I don't totally understand when it overflows vs. just wraps to the next line). More detail [here](https://github.com/code-dot-org/code-dot-org/pull/58449#discussion_r1593089650)

Reply:
If there's a way to force the example in that comment to wrap that would also work

Reply:
A `white-space: nowrap` style we have applied to `code` HTML elements seems to prevent it from wrapping...I could potentially override that styling here (see screenshot where I've disabled it) rather than allowing the content to scroll? But a) I'm not sure if this example is the only situation where we'd have content coming back from the model that won't wrap, and b) I kinda like the scroll experience in this case better than the wrap?

<img width="1680" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/25372625/0ec49e22-27c1-4579-b8d4-e1b9099e191d">

Reply:
Gotcha, yeah this is fine for now then